packages/circuit-ui/components/ImageInput/ImageInput.tsx

```diff
@@ -79,6 +80,10 @@ export interface ImageInputProps
    * An information or error message, displayed below the input.
    */
   validationHint?: string;
+  /**
+   * The border-radius of the image component. Default `peta`.
+   */
+  borderRadius?: keyof Theme['borderRadius'];
```

Comment:
Should we really allow all border-radii here? Or should it be just `peta` and `pill`? (we can pick them from the theme)

Reply:
As we discussed on Slack, I had allowed all border-radii to make the ImageInput compatible with any image component, not just the Avatar component. This is now moot since we agreed on a different approach and I've removed the prop again.